Alerting: Remove `ngalert` feature toggle and introduce two new settings for enabling Grafana 8 alerts and disabling them for specific organisations (#38746)
grafana/grafana: docs/sources/alerting/unified-alerting/_index.md

# Overview of Grafana 8 alerts

> Grafana 8.0 has new and improved alerts that centralizes alerting information for Grafana managed alerts as well as alerts from Prometheus-compatible data sources into one user interface and API. 

Reviewer: ```suggestion
Grafana 8.0 has a new and improved alerting sub-system that centralizes alerting information for Grafana managed alerts and alerts from Prometheus-compatible data sources into one user interface and API. 
```

Alerts allow you to know about problems in your systems moments after they occur. Robust and actionable alerts help you identify and resolve issues quickly, minimizing disruption to your services.

> > **Note:** Starting from the 8.2, out of the box, Grafana 8 alerts are enabled by default.

Reviewer 2: ```suggestion
> **Note:** Out of the box, Grafana 8 alerts are enabled. You can optionally enable legacy alerts by updating your Grafana configuration. 
```
